### PR TITLE
Updated the copyright year to 2018.

### PR DESCRIPTION
**Pull request by hjafarpour**

### CODE REVIEW

File: ksql-common/src/main/java/io/confluent/ksql/util/WelcomeMsgUtils.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public static void displayWelcomeMessage(
         "==========================================="
     };
 
-    final String copyrightMsg = "Copyright 2017 Confluent Inc.";
+    final String copyrightMsg = "Copyright 2018 Confluent Inc.";
```
Comment (dguy) [inline ksql-common/src/main/java/io/confluent/ksql/util/WelcomeMsgUtils.java]: I believa @blueedgenick mentioned that this should probably be "Copyright 2017-2018 ..."

Comment (big-andy-coates) [reply ksql-common/src/main/java/io/confluent/ksql/util/WelcomeMsgUtils.java]: As Damian says, @blueedgenick requested I revert my change when I made the same change:

https://github.com/confluentinc/ksql/pull/806#issuecomment-368953235

Comment (apurvam) [reply ksql-common/src/main/java/io/confluent/ksql/util/WelcomeMsgUtils.java]: Ah.. Yes. It looks like that's the practice for most products, like the copyright notice on my Mac: 
screen shot 2018-03-07 at 11 25 13 am

**Comment by hjafarpour** [comment]
OK, I'll change it to a range as suggested by @apurvam  and @blueedgenick .

**Pull request closed without merging**